codecov.yml

```diff
@@ -0,0 +1,12 @@
+coverage:
+  status:
+    project:
+      default:
+        enabled: yes
+        target: auto
+    patch:
+      default:
+        enabled: yes
+        target: 100%
```

Comment:
@e-kolpakov try getting rid of the percentage sign. You want something like https://github.com/edx/course-discovery/blob/master/codecov.yml. You should probably disable comments, too; they're noisy.